src/main/java/ro/societateahermes/backendservice/entities/User.java

```diff
@@ -24,4 +30,24 @@
     private String field;
     private String language;
 
+    @OneToMany(mappedBy = "user")
+    private List<Participation> listOfParticipation;
+
+
+    public User(String firstName,String lastName,String email,String username,String password,String university)
```

Comment:
Ai pus un @AllArgsConstructor. nu ai nevoie de un alt constructor. Cand folosesti adnotarea aia iti creaza un constructor cu toate fieldurile entitatii. Ele trebuiesc date in ordinea declararii. Adica daca ai declarat fieldurile in ordinea asta: String name; String facebook; Long id; Constructorul generat va arata asa: public MyEntity(String name, String facebook, Long id)